### PR TITLE
Guest registration

### PR DESCRIPTION
**Pull request by townsend3721**

### CODE REVIEW

File: app/controllers/guests_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class GuestsController < ApplicationController
+	def new
+		@guest = Guest.new
+		@guests = Guest.all
+		@all_events = Event.all
+	end
+	def create
+		params.permit!
+		@guest = Guest.new(params[:guest])
+		if @guest.firstname.blank?
```
Comment (harperiscoding) [inline app/controllers/guests_controller.rb]: these checks should all be done through model validations.

File: app/helpers/guests_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module GuestsHelper
```
Comment (harperiscoding) [inline app/helpers/guests_helper.rb]: delete unused file

File: app/models/guest.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+class Guest < ApplicationRecord
```
Comment (harperiscoding) [inline app/models/guest.rb:1]: no relation

**Review by harperiscoding — CHANGES_REQUESTED** [review]

**Review by harperiscoding — APPROVED** [review]

File: app/controllers/guests_controller.rb [hunk 4]
```diff
@@ -0,0 +1,44 @@
+class GuestsController < ApplicationController
+  before_action :set_guest, only: [:destroy]
+  before_action :set_event
+
+
```
Comment (nicolepb) [inline app/controllers/guests_controller.rb]: I'm not sure why it's indented weirdly, it's a little confusing, maybe just fix this

**Review by nicolepb — CHANGES_REQUESTED** [review]
this is better, just probably clean the indents in guests_controller so it's more readable

**Pull request merged**